conviva/src/main/java/com/bitmovin/analytics/conviva/PlayerAdapter.java

```diff
@@ -0,0 +1,32 @@
+package com.bitmovin.analytics.conviva;
+
+import com.bitmovin.analytics.conviva.helper.WithEventEmitter;
+import com.conviva.sdk.ConvivaSdkConstants;
+
+import java.util.HashMap;
+
+public interface PlayerAdapter {
```

Comment:
@strangesource I stuck to your initial naming. However, I'm not sure if this is strictly speaking an `Adapter` 🤔 IMO it's kinda a `Decorator`. But I don't think it's a big deal and I also don't know what's best practice in Java for something like this. I guess in `Kotlin` it would have been a simple extension 🤔

Reply:
You are right, this is not an adapter. Let's rename to decorator pls. 👍

Reply:
[8f54124](https://github.com/bitmovin/bitmovin-player-android-analytics-conviva/pull/79/commits/8f5412437cda0b5f5d819ea8cc15b100ef901941)

Reply:
Do you think, after this change, it makes sense to rename the instance variable from `playerDecorator` to just `player` everywhere? 🤔

Reply:
Yes, makes sense. 👍

Reply:
[1988b97](https://github.com/bitmovin/bitmovin-player-android-analytics-conviva/pull/79/commits/1988b972175b375a2bea1932f754b5ef689e559b)